site/docs/components/progress/usage.mdx

```diff
@@ -0,0 +1,43 @@
+---
+# Leave the frontmatter as is
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  label: Usage
+data:
+  $ref: ./#/data
+---
+
+### Using the Component Name component
+
+It should be used when the operation will take more than a second to complete, to reassure users that the operation is in progress.
+
+#### When not to use Component Name
+
+Do not use when an operation takes a short amount of time, as it may be confusing and distracting for users.
+
+### Import
+
+{/* Update the text and code snippet below as needed: */}
+
+To import Component Name from the core Salt package, use:
+
+```js
+import { CircularProgress } from "@salt-ds/lab";
+// or
+import { LinearProgress } from "@salt-ds/lab";
```

Comment:
```suggestion
import { CircularProgress } from "@salt-ds/core";
// or
import { LinearProgress } from "@salt-ds/core";
```
assuming this isnt going in until Progress is in core, re the PR description and above comment

Reply:
Good spot, thanks!